test/fixture/nuxt.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
We may use esm exports and also directly import module (`import yandex from '../../src/module'`) to ensure having proper coverage